Common/DeviceSchema/CommandHistorySchemaHelper.cs

```diff
@@ -14,12 +11,6 @@ namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.DeviceSch
     /// </summary>
     public static class CommandHistorySchemaHelper
     {
-        public const string RESULT_PENDING ="Pending";
```

Comment:
I had the same question so I grepped these strings, they're actually not used in anywhere. The command status strings are coming from resx files